### PR TITLE
Add SPDX header - batch #1

### PR DESCRIPTION
**Pull request by AndyScherzinger**
* Resolves: noid

## Summary

* first batch of files with SPDX headers added
* ensured all names are wihtin the authors' file
* renamed some files to markdown (authors and copying-readme) for better formatting
* added a dep5 file with definitions for all translations.

This PR brings us to
* files with copyright information: 8295 / 10990
* files with license information: 4243 / 10990

## TODO

- [ ] review
- [ ] create further PRs after each SPDX PR got merged, to get to 100% in batches - in the end for every file in the repo there will be a copyright and license info present.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
also looping in @skjnldsv @Altahrim @blizzz for https://github.com/nextcloud/server/pull/45244/commits/af73ed071ab4a11e09b4abaff2cabad5ceceded1 - just in case more is needed from a release/packaging perspective

File: build/psalm-baseline.xml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (nickvergessen) [inline build/psalm-baseline.xml:2]: this file is generated. need to put into dep5

File: build/psalm-baseline-ocp.xml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```
Comment (nickvergessen) [inline build/psalm-baseline-ocp.xml:2]: this file is generated. need to put into dep5

File: build/psalm/AppFrameworkTainter.php [hunk 3]
```diff
@@ -1,28 +1,9 @@
...
- * copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- * SOFTWARE.
+ * SPDX-FileCopyrightText: 2020 Lukas Reschke <lukas@statuscode.ch>
```
Comment (nickvergessen) [inline build/psalm/AppFrameworkTainter.php:4]:
```suggestion
 * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
```

File: build/stubs/IntlChar.php
Comment (nickvergessen) [inline build/stubs/IntlChar.php:1]: we should exclude stubs for the moment. I'm not sure if generating them changes the copyright

File: core/css/animations.scss [hunk 4]
```diff
@@ -1,22 +1,6 @@
 /*!
```
Comment (nickvergessen) [inline core/css/animations.scss:1]:
```suggestion
/**
```

Comment (AndyScherzinger) [reply core/css/animations.scss:1]: This way the comment would be lost in the css version, hence I would keep it this way.

Comment (nickvergessen) [reply core/css/animations.scss:1]: All other scss files are like this, it's a typo in here.
If it looses the copyright in the compiled asset we should add `core/css/*.css` to the dep5

Comment (AndyScherzinger) [reply core/css/animations.scss:1]: Alright, fair enough 👍

File: build/stubs/IntlChar.php
Comment (AndyScherzinger) [reply build/stubs/IntlChar.php:1]: Done, https://github.com/nextcloud/server/pull/45244/commits/e963d940af0779bf2145ba74873bded25db6a0b3

**Comment by AndyScherzinger** [comment]
Resolved all your comments @nickvergessen 👍 Thanks!

**Review by nickvergessen — APPROVED** [review]

**Comment by blizzz** [comment]
> also looping in @skjnldsv @Altahrim @blizzz for [af73ed0](https://github.com/nextcloud/server/commit/af73ed071ab4a11e09b4abaff2cabad5ceceded1) - just in case more is needed from a release/packaging perspective

https://github.com/nextcloud/updater/blob/master/lib/Updater.php

Keep in mind the the next upgrade will always work with the already shipped updater!

**Review by susnux — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
@blizzz 

> @AndyScherzinger during release building, we delete COPYING-README from bundled apps. Should this stay?

No, can still be removed.

> https://github.com/nextcloud/updater/blob/master/lib/Updater.php#L206-L257
> Keep in mind the the next upgrade will always work with the already shipped updater!

I'll create a PR, will need a review since I can't say if expected files which are missing would be an issue.

**Comment by AndyScherzinger** [comment]
I reverted the renaming of the root folder files, to keep the [changeset for the updater](https://github.com/nextcloud/updater/pull/555) small(er) #45250

**Pull request merged**